### PR TITLE
Preparing for rc0

### PR DESCRIPTION
**Pull request by Kirill888**
## Code

- Adding `GeoBox` construction from bounding box
- Adding visualization things, svg output, html repr output
- Adding various misc convenience bits that were missing and were needed when developing visualization parts
- Adding geometry transform with affine matrix
- Adding data folder with geojson of oceans and css for geobox html display
- Minor fix in geobox slicing when single pixel row/column is selected

## Docs

- better overview
- update conda install instructions

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = odc-geo
+name = odc_geo
```
Comment (gadomski) [inline setup.cfg]: Why the name change?

Comment (Kirill888) [reply setup.cfg]: warning from `pip` in python the two are equivalent anyway.

Comment (gadomski) [reply setup.cfg]: From which command? I just tried `pip install -e .` and `python -m build` and neither gave me a warning. IMO it feels better to have the name match the github slug?

Comment (Kirill888) [reply setup.cfg]: actually, I'm not longer sure if I was reading the right warning, I was failing to include data into packaging at the time, let's try it again with the previous name.

Comment (Kirill888) [reply setup.cfg]: @gadomski warning was coming from `options.package_data` not from package name, I have reverted package name change and fixed package_data section.

**Review by gadomski — APPROVED** [review]

**Pull request merged**